File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    max_height = max_width = 0
    for size in sizes:
        width = max(size)
        max_width = max(max_width,width)
        
        height = min(size)
        max_height = max(max_height, height)
        
        
    return max_width*max_height

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    var answer = 0;
    let max_height = 0
    let max_width = 0
    for (size of sizes){
        let width = Math.max(size[0],size[1])
        max_width = Math.max(max_width, width)
        
        let height = Math.min(size[0],size[1])
        max_height = Math.max(max_height, height)
    }
    return max_width*max_height
}
